package.json

```diff
@@ -63,13 +63,13 @@
     "@reason-native/pastel": "*",
     "@reason-native/rely": "*",
     "@esy-ocaml/reason": "*",
-    "ocaml": "~4.7.0"
+    "ocaml": "~4.7.0",
```

Comment:
What about bumping Ocaml version to 4.8 ?

Reply:
Yes, we could do that. Perhaps match hello-reason in this regard

Reply:
Done :)